app/api.test.js

```diff
@@ -40,6 +40,35 @@ describe('api.js', function () {
         })
     })
   })
+  describe('resolveName', function () {
+    it('should reject when IPFS is not started', function () {
+      // arrange
+      api.setClientInstance(null)
+      // act
+      return api.resolveName()
+        .catch(err => {
+          // assert
+          expect(err).toBe(ERROR_IPFS_UNAVAILABLE)
+        })
+    })
+
+    it('should resolve the given name', function () {
```

Comment:
Should we add a case when the IPNS is not available?

Reply:
We don't catch the error though, so we wouldn't be testing the implementation. Should we catch `.name.resolve` and give an custom error?

Reply:
It is fine for now